server/endpoint/creator/aws/aws.go

```diff
@@ -0,0 +1,10 @@
+package aws
+
+type Roles struct {
```

Comment:
I would prefer to rename it to AWSRoles instead, but creating package is fine also.

Reply:
done